src/instance.ts

```diff
@@ -319,15 +447,18 @@ class Instance extends common.ServiceObject {
             method: 'deleteInstance',
             reqOpts,
           },
-          (err, resp) => {
+          (err: Error, resp: LongrunningIOperation) => {
```

Comment:
The `apiResponse` type has `IEmpty` listed within the `DeleteInstanceCallback` interface, this appears to conflict with that.